src/protocol.h

```diff
@@ -316,6 +332,14 @@ enum
     // collisions and other cases where nodes may be advertising a service they
     // do not actually support. Other service bits should be allocated via the
     // BIP process.
+
+    // BUIPXXX - Graphene blocks - begin section
+    // NODE_GRAPHENE means the node supports Graphene blocks
+    // If this is turned off then the node will not service graphene requests nor
+    // make graphene requests
+    NODE_GRAPHENE = (1 << 6),
```

Comment:
in case we are going to port graphene to the BU legacy code base (`BitcoinCore` branch) guess we need to check if some other implementation is already using this bit to signal some other services.

Reply:
I've made a note about this. I will be sure to double-check before merging.

Reply:
great